### PR TITLE
🔧 migrate b-button part 2

### PR DESCRIPTION
**Pull request by roiLeo**
part of #5852 

- replaced all `b-button`
- removed unused component

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -38,9 +38,9 @@
                 ref="mobilSearchRef"
                 hide-filter
                 class="is-flex-grow-1 mt-3" />
-              <b-button class="cancel-btn" @click="hideMobileSearchBar">
+              <a class="cancel-btn p-3" @click="hideMobileSearchBar">
```
Comment (daiagi) [inline components/Navbar.vue:41]: I think this will be better as a `button` rather than `a`

Other then that... lgtm

**Review by daiagi — APPROVED** [review]

**Comment by yangwao** [comment]
image

👀

**Comment by yangwao** [comment]
@roiLeo let's fix it :)

**Comment by roiLeo** [comment]
ooo

**Pull request merged**